fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/cascades/VariadicFunctionValueTest.java

```diff
@@ -232,41 +280,53 @@ public Stream<? extends Arguments> provideArguments(final ExtensionContext conte
                     Arguments.of(List.of(DOUBLE_3, DOUBLE_3), new VariadicFunctionValue.CoalesceFn(), 3.0, false),
                     Arguments.of(List.of(STRING_3, STRING_3), new VariadicFunctionValue.CoalesceFn(), "c", false),
                     Arguments.of(List.of(BOOLEAN_2, BOOLEAN_2), new VariadicFunctionValue.CoalesceFn(), true, false),
+                    Arguments.of(List.of(LIST_INT_1, LIST_INT_1), new VariadicFunctionValue.CoalesceFn(), List.of(1, 2, 3), false),
+                    Arguments.of(List.of(RECORD_1, RECORD_1), new VariadicFunctionValue.CoalesceFn(), buildMessageForRecordType(List.of("a", 1, 1.0f)), false),
```

Comment:
could you please put this `buildMessageForRecordType(List.of("a", 1, 1.0f))` in a method?

Reply:
done